custom_components/tahoma/sensor.py

```diff
@@ -22,6 +22,14 @@
 from .const import DOMAIN
 from .entity import OverkizDescriptiveEntity, OverkizSensorDescription
 
+OVERKIZ_BATTERY_TO_HOME_ASSISTANT = {
```

Comment:
That’s a really long name :)

```suggestion
OVERKIZ_BATTERY_TO_HA = {
```